digits/scheduler.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, gpu_list=None, verbose=False):
         gpu_list -- a comma-separated string which is a list of GPU id's
         verbose -- if True, print more errors
         """
-        self.jobs = []
+        self.jobs = dict()
```

Comment:
I was thinking that dict() was sorted associative container.  I was wrong.  It doesn't look like python has that.  The OrderedDict is insertion order, which should be sorted at red time and job add time.